doc/user/user/project/index.md

```diff
@@ -231,14 +269,56 @@ Start by [viewing the project members](#project-members).
 
 To remove a project member, click on the remove button on the right-hand side of the table:
 
-![Remove project member button.](images/remove-project-member-button.png)
+![Remove project member button.](images/remove-project-member.png)
 
 When you click the remove button, you will be asked to confirm the project member removal:
 
 ![Remove project member confirmation dialog.](images/remove-project-member-confirm.png)
 
 To confirm, click the "Ok" button.
 
+#### Removing a user group from a project
+
+You may want to completely remove all permissions for a user group to access data in a project. To remove those permissions, you must remove the user group from the project user groups list.
+
+Start by [viewing the project user groups](#project-user-groups).
+
+To remove a project user group, click on the remove button on the right-hand side of the table:
+
+![Remove project user group button.](images/remove-project-usergroup.png)
+
+When you click the remove button, you will be asked to confirm the project member removal:
+
+![Remove project user group confirmation dialog.](images/remove-project-usergroup-confirm.png)
+
+To confirm, click the "Ok" button.
+
+### Metadata
+
+Metadata associated with the project can be viewed from the **Metadata** tab. A project **Manager** can view **all** the metadata fields associated with the project, and view/modify the metadata restriction levels for each of these fields.
+
+![Metadata fields manager view](images/metadata-fields-manager.png)
+
+The **Manager** also has the ability to select metadata fields, create templates, and set a default template which members of the project can use to view data on the Linelist page.
+
+![Metadata templates manager view](images/metadata-templates-list-manager.png)
+
+Clicking on the name of the template takes you to a page where you can modify template details. Note that the field restriction levels must be modified from the **Fields** tab and cannot be modified within a template.
+
+![Metadata template fields manager view](images/metadata-template-fields-list-manager.png)
+
+A collaborator can only view a subset of the metadata fields which are at their metadata restriction level and below. For example the user below has a project metadata role of Level 2, so they can only view metadata fields which have a restriction Level 1 or Level 2:
+
+![Metadata fields collaborator view](images/metadata-fields-collaborator.png)
+
+A project collaborator can only view/download the templates that exist and the fields that makes up those templates:
+
+![Metadata templates  collaborator view](images/metadata-templates-list-collaborator.png)
+
+Clicking on the name of the template takes you to a page where you can view template details.
+
+![Metadata template fields collaborator view](images/metadata-template-fields-list-collaborator.png)
```

Comment:
Can you add a single line space on line 328.  The text is wrapping poorly around the image without it.

Reply:
Added in c306839c050f21001326c89da18632a7075d918b